extras/userdoc/md/documentation/part-2-populations-of-neurons.md

```diff
@@ -115,6 +120,42 @@ Note that we are being rather lax with random numbers here. Really we have to
 take more care with them, especially if we are using multiple threads or
 distributing over multiple machines. We will worry about this later.
 
+## Generating populations of neurons with deterministic connections
+
+In the previous handout two neurons were connected using synapse 
+specifications. In this section we extend this example to two populations of 
+ten neurons each.
+
+    import pylab
+    import nest
+    pop1 = nest.Create("iaf_neuron", 10)
+    nest.SetStatus(pop1, {"I_e": 376.0})
+    pop2 = nest.Create("iaf_neuron", 10)
+    multimeter = nest.Create("multimeter", 10)
+    nest.SetStatus(multimeter, {"withtime":True, "record_from":["V_m"]})
+
+If no connectivity pattern is specified, the populations are connected via 
+the default rule, namely `all\_to\_all`. Each neuron of `pop1` is connected 
+to every neuron in `pop2`, resulting in $10^2$ connections.
```

Comment:
@abigailm: do you mean on GitHub or when you generate the html? GitHub does not use mathjax...